packages/react-components/src/components/DateInput.tsx

```diff
@@ -0,0 +1,29 @@
+import React from 'react';
```

Comment:
TextArea comments apply here also

Reply:
please implement displayFormat, editFormat props

Reply:
This task is not picked yet, it will be done incrementally!